scripts/deployment/goerli-testnets.json

```diff
@@ -69,28 +69,28 @@
       ]
     },
     {
-      "name": "Boba Goerli",
-      "rpc": "https://goerli.boba.network/",
-      "chain_id": 2888,
-      "l1_messenger": [
-        "OptimismL1Messenger",
-        "0xA6fA0867F39f3A3af7433C8A43f23bf26Efd1a48"
-      ],
-      "l2_messenger": "OptimismL2Messenger",
-      "finality_period": 1800,
+      "name": "Goerli",
+      "rpc": "https://goerli.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161",
```

Comment:
Are we okay with this?

Reply:
i think we shouldnt put rpc to here

Reply:
or just use an official one like we are doing in run-your-own agent as example

Reply:
then it is ok for me